docs/OPTIONS-LIST.md

```diff
@@ -199,6 +199,10 @@ Below are the "only" options that only have a general option
 
   ```--unrankedonly```  Only accept unranked matches
 
+  ```--privateonly```  (**Not yet supported**) Only accept private matches
+
+  ```--publiconly```  (**Not yet supported**) Only accept public (non-private) matches
```

Comment:
If this is not supported, we should not add it.

Reply:
@anoek said that he may take some time in the future to add it in notification, and several bot admins said that they wanted to use this feature

if this feature is added soon in the future, it will be easier to toggle it on